carbon-design-system/carbon-platform
# feat: added esc working on focus

Closes #163 

## Testing: 

-  Make sure you can close Popover using ESC
- Make sure focus  moves to popover on open
- Make sure query state is updated so that the popover remains open each time a new tag is selected
